internal/xsql/sqlValidator.go

```diff
@@ -115,3 +115,69 @@ func isSRFExists(node ast.Node) bool {
 	})
 	return exists
 }
+
+func validateFields(stmt *ast.SelectStatement) {
```

Comment:
Not only fields/sortFields has the fieldRef. For example, where a.b > 10. Need to visit the full statement and update all fieldRef